### PR TITLE
Unbox <links> for ;;play

### PR DESCRIPTION
**Pull request by natanbc**
See #219

### CODE REVIEW

File: FredBoat/src/main/java/fredboat/command/music/control/PlayCommand.java [hunk 1]
```diff
@@ -99,11 +99,16 @@ public void onInvoke(CommandContext context) {
         }
 
         //Search youtube for videos and let the user select a video
-        if (!args[1].startsWith("http")) {
+        if (!(args[1].startsWith("http") || (args[1].startsWith("<http") && args[1].endsWith(">")))) {
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/music/control/PlayCommand.java]: Why not remove the <> from the string first, then use that instead?

**Comment by napstr** [comment]
Any update on this @natanbc ?

**Comment by SuhanKoh** [comment]
Is anyone taking over this issue? or should it be closed?

**Comment by Shredder121** [comment]
@napstr How's this now?

**Pull request merged**